AI/Nullkiller/AIUtility.h

```diff
@@ -48,19 +48,25 @@
 #include "../../lib/mapObjects/CObjectHandler.h"
 #include "../../lib/mapObjects/CGHeroInstance.h"
 #include "../../lib/CPathfinder.h"
+#include "../../CCallback.h"
 
 #include <chrono>
 
 using namespace tbb;
 
-class CCallback;
-class Nullkiller;
-struct creInfo;
+VCMI_LIB_NAMESPACE_BEGIN
 
 typedef const int3 & crint3;
 typedef const std::string & crstring;
 typedef std::pair<ui32, std::vector<CreatureID>> dwellingContent;
 
+VCMI_LIB_NAMESPACE_END
```

Comment:
not sure why this is needed - `using namespace` is already in StdInc.h

Reply:
oh, it's a header - here it's definitely needed

Reply:
Refactored a bit